app/code/Magento/Captcha/Observer/CheckUserLoginObserver.php

```diff
@@ -135,7 +135,7 @@ public function execute(\Magento\Framework\Event\Observer $observer)
             : null;
         if ($captchaModel->isRequired($login)) {
```

Comment:
According to your description, the problem is in this check `if ($captchaModel->isRequired($login)){ ...` It should return `false` if captcha is turned off for login page. So, we need to consider fixing this condition instead of checking the `$word`. Correct me if I'm wrong. Thank you.

Reply:
I mean even I'm turned on the captcha for login page, and still show that message when I first time login (this time Will not show the captcha on this page),

Reply:
@GlennCheng in this case we should add some check that captcha is not mandatory this time.
Your solution will disable capcha for all the times

Reply:
@ihor-sviziev Thanks, i pushed another one commit, please check that.